File: 프로그래머스/unrated/133499. 옹알이 （2）/옹알이 （2）.py
def solution(babbling):
    count = 0
    words = [ "aya", "ye", "woo", "ma" ]
    for b in babbling:
        for word in words:
            if word * 2 not in b:
                b = b.replace(word, ' ')
        if b.strip() == '':
            count += 1
    return count

File: 프로그래머스/1/133499. 옹알이 （2）/옹알이 （2）.java
class Solution {
    public int solution(String[] babbling) {
        String[] words = {"aya", "ye", "woo", "ma"};
        int total = 0;
        // 연속된 같은 단어 X, 단어가 목록에 있어야 함
        for (String b : babbling){
            for (String word : words){
                if (b.indexOf(word+word)!=-1){
                    break;
                }
                b=b.replace(word,"0");
            }
            if (b.matches(".*[a-z].*")){
                continue;
            }
            total+=1;
        }
        return total;
    }
}